<launch>
<!-- Launch file for Livox AVIA LiDAR -->

	<arg name="rviz" default="true" />

	<rosparam command="load" file="$(find m_detector)/config/kitti0.yaml" />

	<arg name="file_path" default=""/>	
	<arg name="out_path" default=""/>	
	<arg name="out_origin_path" default=""/>
	<arg name="pcd_way" default="/home/weihairuo/bag/kitti_with_loop"/>
	<arg name="id" default="02"/>

	<!-- <node pkg="fast_lio" type="generate_odom" name="generate_odom" output="screen" />  -->

	<node pkg="m_detector" type="detector_with_fastlio" name="detector_with_fastlio" output="screen"/>
	<param name="point_filter_num" type="int" value="1"/>
	<param name="max_iteration" type="int" value="4" />
	<param name="filter_size_surf" type="double" value="0.5" />
	<param name="filter_size_map" type="double" value="0.5" />
	<param name="cube_side_length" type="double" value="5000" />

	<param name="dyn_obj/out_file" type="string" value="$(arg out_path)" />
	<param name="dyn_obj/out_file_origin" type="string" value="$(arg out_origin_path)" />
	<param name="dyn_obj/time_file" type="string" value="$(arg file_path)" />

	<param name="dyn_obj/save_file" type="string" value="$(arg pcd_way)/$(arg id)/steady/" />
	<param name="dyn_obj/m_detector_file" type="string" value="$(arg pcd_way)/$(arg id)/" />
     
    launch-prefix="gdb -ex run --args"
	<!--  <node pkg="fast_lio" type="revise_kitti" name="revise_kitti" output="screen" /> -->
 
	<!-- <node pkg="fast_lio" type="replay_hist" name="replay_hist" output="screen" /> -->
  
<!--	<group if="$(arg rviz)">-->
<!--	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find m_detector)/rviz_cfg/demo.rviz" />-->
<!--	</group>-->
launch-prefix="gdb -ex run --args"
  	 
 
</launch>